### PR TITLE
DOC: Clarified pandas_version in to_json

### LINKED ISSUE #26637: Improve docs for pandas_version in Dataframe to_json(orient='table')
#### Code Sample, a copy-pastable example if possible

```python
# Your code here
import pandas as pd
df = pd.DataFrame({'A': [1,2,3], 'B': [4,5,6], 'C': [7,8,9] })
df.to_json(orient='table', index='False')
```
#### Problem description
I was attempting to serialize a dataframe and omit the indexes.
In the generated JSON the indexes remained and the pandas_version value in the JSON was "0.20.0"

The results I obtained on Windows 10 and MacOS 10.14.5 was:
{"schema": {"fields":[{"name":"values","type":"integer"},{"name":"A","type":"integer"},{"name":"B","type":"integer"},{"name":"C","type":"integer"}],"primaryKey":[null]**,"pandas_version":"0.20.0"}**, "data": [{"index":0,"A":1,"B":4,"C":7},{"index":1,"A":2,"B":5,"C":8},{"index":2,"A":3,"B":6,"C":9}]}'

[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]

**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates!

**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.

For documentation-related issues, you can check the latest versions of the docs on `master` here:

https://pandas-docs.github.io/pandas-docs-travis/

If the issue has not been resolved there, go ahead and file it in the issue tracker.

#### Expected Output
{"schema": {"fields":[{"name":"values","type":"integer"},{"name":"A","type":"integer"},{"name":"B","type":"integer"},{"name":"C","type":"integer"}],"primaryKey":[null],"pandas_version":"0.24.2"}, "data": [{"A":1,"B":4,"C":7},{"A":2,"B":5,"C":8},{"A":3,"B":6,"C":9}]}'
#### Output of ``pd.show_versions()``

<details>

[paste the output of ``pd.show_versions()`` here below this line]
INSTALLED VERSIONS
------------------
commit: None
python: 3.7.3.final.0
python-bits: 64
OS: Windows
OS-release: 10
machine: AMD64
processor: Intel64 Family 6 Model 69 Stepping 1, GenuineIntel
byteorder: little
LC_ALL: None
LANG: en_US.UTF-8
LOCALE: None.None

pandas: 0.24.2
pytest: 4.3.1
pip: 19.0.3
setuptools: 40.8.0
Cython: 0.29.6
numpy: 1.16.2
scipy: 1.2.1
pyarrow: None
xarray: None
IPython: 7.4.0
sphinx: 1.8.5
patsy: 0.5.1
dateutil: 2.8.0
pytz: 2018.9
blosc: None
bottleneck: 1.2.1
tables: 3.5.1
numexpr: 2.6.9
feather: None
matplotlib: 3.0.3
openpyxl: 2.6.1
xlrd: 1.2.0
xlwt: 1.3.0
xlsxwriter: 1.1.5
lxml.etree: 4.3.2
bs4: 4.7.1
html5lib: 1.0.1
sqlalchemy: 1.3.1
pymysql: None
psycopg2: None
jinja2: 2.10
s3fs: None
fastparquet: None
pandas_gbq: None
pandas_datareader: None
gcsfs: None
</details>

### PR DESCRIPTION
**Pull request by tnwei**
- [X] closes #26637
- [ ] tests added / passed
- [X] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -2207,6 +2207,10 @@ def to_json(
 
                 Describing the data, where data component is like ``orient='records'``.
 
+                Note that ``orient='table'`` contains a 'pandas_version' field under
+                'schema'. This denotes the version of `pandas` where the schema was last
```
Comment (arw2019) [inline pandas/core/generic.py:2211]: `denotes` &#8594; `stores`

Comment (arw2019) [inline pandas/core/generic.py:2211]: `where the schema was last revised` &#8594; `used in the latest revision of the schema`

**Review by arw2019 — COMMENTED** [review]
Thanks @tnwei for the PR!

**Comment by tnwei** [comment]
Thanks @arw2019 for the review, edited wording as suggested

**Review by arw2019 — APPROVED** [review]

File: pandas/core/generic.py [hunk 2]
```diff
@@ -2207,6 +2207,10 @@ def to_json(
 
                 Describing the data, where data component is like ``orient='records'``.
 
+                Note that ``orient='table'`` contains a 'pandas_version' field under
```
Comment (jreback) [inline pandas/core/generic.py:2210]: can you move this to the Notes section (or create if its not there)

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by mroeschke — APPROVED** [review]

File: pandas/core/generic.py (hunk 2 above)
Comment (tnwei) [reply pandas/core/generic.py]: Done!

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @tnwei

**Comment by tnwei** [comment]
Thanks for merging @jreback . Hacktoberfest brought me here, would you mind adding the `hacktoberfest-accepted` label to this PR? TQVM!

**Comment by jreback** [comment]
> Thanks for merging @jreback . Hacktoberfest brought me here, would you mind adding the `hacktoberfest-accepted` label to this PR? TQVM!

no problem

**Pull request merged**